src/pilot/inspect.patch

```diff
@@ -0,0 +1,70 @@
+--- inspect.py	2019-03-18 04:26:06.117996201 -0400
```

Comment:
@cdearborn commented previously:

"This looks good, but a new version (z5) of RHOSP has just shipped. This version has the CPU count fix in it. As a result, you will need to regenerate the patch below with the latest shipped version of inspect.py, and that patch should not contain the CPU fix since the fix is already present."

While I have not put my eyes on z5, my impression is this entire file, `src/pilot/inspect.patch`, is not needed.

Have you had an opportunity to look at z5 and confirm it is not needed? If so, please remove it from this PR.

Reply:
@rpioso  Agree

Here what we found after installing undercloud and overcloud. In director inspect.py has only calculate_cpus() method. I think we need this patch for current fix, when pxe_enable code will be available in RHOSP, then we can remove this change.

Reply:
yes, this patch will still be needed for the pxe_enable fix.  The section of the patch that patches in the code for the cpu count fix should be removed since that code is already in OSP13z5

Reply:
Ah, I see that you've already removed the patch for the cpu count fix.  Thanks!

Reply:
I put my eyes on z5 and agree that this patch is needed.